src/colvarcomp_rotations.cpp

```diff
@@ -133,14 +133,25 @@ void colvar::orientation::calc_gradients()
 void colvar::orientation::apply_force(colvarvalue const &force)
 {
   cvm::quaternion const &FQ = force.quaternion_value;
+  auto const rot_inv = atoms->rot.inverse().matrix();
```

Comment:
I think it is better to move this line into the `if (atoms->is_enabled(f_ag_rotate)) {}` case.